lib/fastlane/plugin/wpmreleasetoolkit/helper/ios/ios_l10n_helper.rb

```diff
@@ -196,7 +196,7 @@ def find_diffs(input_dir:, base_lang:, only_langs: nil)
         # Returns true if the file only contains empty lines, i.e. lines that only contains whitespace (space, tab, CR, LF)
         def only_empty_lines?(file)
           File.open(file) do |f|
-            while line = f.gets
+            while (line = f.gets)
```

Comment:
My Ruby might not be up-to-par, but I'm not sure I understand the difference here 😅 

But if the linter is happy with it, I am too if it's functionally identical

Reply:
That's a fair question. The parentheses are needed here mainly to silence a warning and be explicit that it's not a mistake that we do both affectation `=` … and comparison (interpreting the whole expression as a Boolean / truthiness test).

So the linter wants to be sure we didn't mean `==` and accidentally type `=`. Adding the parentheses achieves this, by making the affectation a sub-expression in parentheses that is then interpreted as a (truthiness) value of its own when interpreted as the condition of the `while`.

You can see it as a more compact way of writing `while (line = f.gets) != nil` – except I _think_ (?) that the linter would not recommend this alternative and tell us that the `!= nil` is implicit and can be removed… so here goes.